xchange-lykke/src/main/java/org/knowm/xchange/lykke/service/LykkeTradeServiceRaw.java

```diff
@@ -0,0 +1,84 @@
+package org.knowm.xchange.lykke.service;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.exceptions.ExchangeException;
+import org.knowm.xchange.lykke.LykkeException;
+import org.knowm.xchange.lykke.LykkeAdapter;
+import org.knowm.xchange.lykke.dto.trade.LykkeLimitOrder;
+import org.knowm.xchange.lykke.dto.trade.LykkeOrder;
+import org.knowm.xchange.lykke.dto.trade.LykkeTradeHistory;
+import org.knowm.xchange.service.trade.params.TradeHistoryParamsAll;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.xml.ws.http.HTTPException;
+import java.io.IOException;
+import java.util.List;
+
+public class LykkeTradeServiceRaw extends LykkeBaseService{
+
+    private Logger logger = LoggerFactory.getLogger(LykkeTradeServiceRaw.class);
+
+    public LykkeTradeServiceRaw(Exchange exchange) {
+        super(exchange);
+    }
+
+    public List<LykkeTradeHistory> getLykkeTradeHistory(TradeHistoryParamsAll paramsAll) throws IOException {
+        try {
+            return lykke.getTradeHistory(
+                    LykkeAdapter.adaptToAssetPair(paramsAll.getCurrencyPair()),
+                    paramsAll.getPageLength(),
+                    apiKey);
+        }catch (LykkeException e){
+            throw new ExchangeException(e.getMessage());
+        }
```

Comment:
*Raw methods should throw out the exchanges native exception
implementation of generic interface methods should wrap them into generic ones like you do here
please read: https://github.com/knowm/XChange/wiki/New-Implementation-Best-Practices#step-7